src/containers/Transactions/Simple/NFTokenMint.jsx

```diff
@@ -26,8 +26,10 @@ const NFTokenMint = (props) => {
       </div>
       <div className="row">
```

Comment:
if you add `flex-wrap` here it will also make it so even more of the url can be shown in case it is even longer.

I have a PR coming up that will remove the need to add `flex-wrap` and will apply it to all simple tab rows.  You can choose to wait for that PR if you want.

Reply:
oh just saw that PR. I can close this one, doesnt seem necessary